pkg/webhook/admission/pod/agent_injector_test.go

```diff
@@ -122,7 +122,7 @@ func TestAgentInjector(t *testing.T) {
 								},
 							},
 							Args: []string{"--enable-puller", "--config-dir", "/mnt/configs", "--model-dir", "/mnt/models"},
-							Env:  []v1.EnvVar{},
+							Env:  []v1.EnvVar{{Name: "SERVING_READINESS_PROBE", Value: "null"}},
```

Comment:
Add the readiness probe in the test input so we can verify here?

Reply:
Sorry Dan, am not sure if I understand you correctly, will ping you in chat.